addons/binding/org.openhab.binding.velux/ESH-INF/i18n/velux_en.properties

```diff
@@ -0,0 +1,37 @@
+#/**
```

Comment:
For English no i18n file should be used, instead these text should be present on their specific locations.

Reply:
Reduced for error messages.